public/index.html

```diff
@@ -0,0 +1,741 @@
+      </section>
+      <section class="a3-home-middle">
+        <svg class="slant" xmlns="http://www.w3.org/2000/svg" version="1.1" width="100%" height="100" viewBox="0 0 100 102" preserveAspectRatio="none">
+          <path d="M0 0 L0 100 L100 0 Z" />
+        </svg>
+        <h1 class="home-h1">GET IT FOR FREE</h1>
+        <p class="home-middle-p">Choose your native platform and download the app FREE!</p>
+        <ul class="home-google-ios">
+          <li><a class="submit-button" href="https://play.google.com/store?hl=en"><span class="icon-Googleplay"></span>  Google Play</a></li>
+          <li><a class="submit-button" href="http://www.apple.com/itunes/charts/free-apps/"><span class="icon-iOS"></span>  iOS App Store</a></li>
+        </ul>
+      </section>
+      <section class="a4-home-bottom">
+      <svg class="slant" xmlns="http://www.w3.org/2000/svg" version="1.1" width="100%" height="100" viewBox="0 0 100 102" preserveAspectRatio="none">
+        <path d="M0 0 L0 100 L100 0 Z" />
+      </svg>
+      <div class="underlined-heading">
+        <h1 class="underlined-heading-text">Some Amazing Features</h1>
+        <div class="underlined-heading-underline"></div>
+      </div>
+      <section class="home-feature">
+      <ul class="home-feature-list">
+        <li>
+        <figure class="home-feature home-feature-laptop">
+          <div class="container-icon">
+            <p class="icon-laptop"></p>
+          </div>
+          <h5 class="home-feature-h5">RESPONSIVE DESIGN</h5>
+          <figcaption>Eleifend morbi velit porttitor sed ipsum imperdiet ac ullamcorper fusce</figcaption>
+        </figure>
+        </li>
+        <li>
+          <figure class="home-feature">
+            <div class="container-icon">
+              <p class="icon-beaker home-feature-icon"></p><!-- beaker icon -->
+            </div>
+            <h5 class="home-feature-h5">EASY TO CUSTOMIZE</h5>
+            <figcaption>Eleifend morbi velit porttitor sed ipsum imperdiet ac ullamcorper fusce</figcaption>
+          </figure>
+        </li>
+        <li>
+          <figure class="home-feature">
+            <div class="container-icon">
+              <p class="icon-airplane  home-feature-icon"></p><!-- airplane icon -->
+            </div>
+            <h5 class="home-feature-h5">PERFECT SHOWCASE</h5>
+            <figcaption>Eleifend morbi velit porttitor sed ipsum imperdiet ac ullamcorper fusce</figcaption>
+          </figure>
+        </li>
+        <li>
+          <figure class="home-feature">
+            <div class="container-icon">
+              <p class="icon-cog home-feature-icon"></p><!-- gear setting icon -->
+            </div>
+            <h5 class="home-feature-h5">APP STORE SUPPORT</h5>
+            <figcaption>Eleifend morbi velit porttitor sed ipsum imperdiet ac ullamcorper fusce</figcaption>
+          </figure>
+        </li>
+      </ul>
+    </section>
+  </section>
+</section>
+  <section class="b1-support-download">
```

Comment:
Here's where I think your naming system starts to break down - what's `b1`? (`a1`, `a2`, and `a3` are similarly confusing as well). In the context of the comp, how would you refer to this section? What is the design attempting to do? When I look at it, I see a focus on something the company is providing, so perhaps it's a `product-upsell` (selling the availability of conference support).